inc/registration.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * Register included providers.
+ *
+ * @param Core_Sitemaps_Provider[] $providers List of registered providers.
+ *
+ * @return Core_Sitemaps_Provider[] Updated list.
+ */
+function core_sitemaps_registration( $providers ) {
+	$providers['sitemap-index'] = new Core_Sitemaps_Index();
```

Comment:
The idea here is that  the key could be the url for the sitemap? or alternatively the index can iterate on the providers and ask for a certain method, which is probably a nice refinement of this PR.

Reply:
Addressed through https://github.com/GoogleChromeLabs/wp-sitemaps/pull/42/commits/349e62ece2983955fdc43f49c53ac672b4c1e28f

Reply:
Would this make more sense to be `posts` rather than `sitemap-posts` as that's what is being registered for each sitemap? i.e. 
```
	public function register_sitemap() {
		$this->registry->add_sitemap( 'posts', '^sitemap-posts\.xml$' );
	}
```